tests/e2e/unified_test.go

```diff
@@ -158,7 +158,15 @@ func testFixturesInSeries(ctx context.Context, t *testing.T, testPause bool, can
 	}
 
 	t.Run("fixtures", func(t *testing.T) {
-		fixtures := resourcefixture.Load(t)
+		// Skip newly added iam/iampartialpolicy for now as they run under TestIAM_AllInSeries
```

Comment:
By "for now", do you plan to do further refactoring of this filter?

Reply:
I was thinking more about when we collapse the two tests. Ideally there's only one TestAllinSeries 😄